src/game/server/netgui.cpp

```diff
@@ -112,8 +90,8 @@ void CNetGui::OnMessage(int MsgID, void *pRawMsg, int ClientID)
 
 void CNetGui::RemoveElement(int ClientID, int Type, int NetGuiElemID)
```

Comment:
I don't completely get what you mean, but the RemoveElement function is to be used by the mod maker. It deletes the specified element from the players screen